src/Be.Vlaanderen.Basisregisters.SnapshotVerifier/DefaultComparisonConfig.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Be.Vlaanderen.Basisregisters.SnapshotVerifier
+{
+    using System.Collections.Generic;
+    using KellermanSoftware.CompareNetObjects;
+    using KellermanSoftware.CompareNetObjects.TypeComparers;
+
+    public sealed class DefaultComparisonConfig
+    {
+        public static ComparisonConfig Get => new()
```

Comment:
`Instance` or something instead of `Get`.

Reply:
Done.